src/foam/u2/DeleteModal.js

```diff
@@ -29,6 +29,7 @@ foam.CLASS({
     { name: 'TITLE', message: 'Delete ' },
     { name: 'CONFIRM_DELETE_1', message: 'Are you sure you want to delete' },
     { name: 'SUCCESS_MSG', message: ' deleted.' },
+    { name: 'SUCCESS_MSG_REQUEST', message: 'An approval request has been created.' },
```

Comment:
What happens when approvals are not configured/enabled (via rules) - delete would be immediate would it not?